src/components/AuthForm.js

```diff
@@ -188,14 +188,41 @@ const authSchemas = {
         password: {"ui:widget": "password"}
       }
     }
-  }
+  },
+  portier: {
+    schema: {
+      ...baseAuthSchema,
+      required: [...baseAuthSchema.required, "email"],
+      properties: {
+        ...baseAuthSchema.properties,
+        email: {
+          title: "Email address",
+          type: "string",
+          format: "email",
+        }
+      }
+    },
+    uiSchema: {
+      authType: {
+        ...baseUISchema.authType,
+        "ui:help": (
+          <span>
+            <b>Note:</b> The
+            <a href="https://github.com/Kinto/kinto-portier">{" kinto-portier "}</a>
```

Comment:
If that's confusing, here's a more direct explanation ([jsfiddle](https://jsfiddle.net/n1k0/nfLxq4v6/)):

![](http://i.imgur.com/4DfJ4Hi.png)